packages/backend-data/src/convert_schema.ts

```diff
@@ -1,31 +1,31 @@
-import {
+import type {
   CustomSqlDataSourceStrategy,
   DataSourceConfiguration,
   DerivedCombinedSchema,
-  DerivedModelSchema,
+  DerivedModelSchema as DerivedDataSchema,
```

Comment:
I think the `DerivedModelSchema` naming is confusing when referring to the type schema object returned from `data-schema`, since the term "Model Schema" has been used historically associated with the string GraphQL schema annotated with Amplify directives (e.g. `@model`). 

We'll open a separate PR with a minor version bump to update the type alias in `'@aws-amplify/data-schema-types'` to `DerivedDataSchema`, then will change this import symbol and remove the import alias.